swift/OnnxRuntimeBindingsTests/SwiftOnnxRuntimeBindingsTests.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import XCTest
+import Foundation
+@testable import OnnxRuntimeBindings
+
+final class SwiftOnnxRuntimeBindingsTests: XCTestCase {
+    let modelPath: String = Bundle.module.url(forResource: "single_add.basic", withExtension: "ort")!.path
+
+    func testGetVersionString() throws {
```

Comment:
previously, the unit tests were all in objectivec/test. it's good to have additional test coverage in Swift, but now it's not so obvious where a new unit test of newly added functionality should be added. should we also add an Objective-C test for ORTVersion?